.werft/installer-tests.ts

```diff
@@ -345,6 +345,7 @@ export async function installerTests(config: TestConfig) {
 
 function runIntegrationTests() {
     werft.phase("run-integration-tests", "Run all existing integration tests");
+    var slackAlerts = new Map<string, string>([])
```

Comment:
```suggestion
    const slackAlerts = new Map<string, string>([])
```
nit: probably shouldn't use `var`

Reply:
I am changing this Map by appending items to it. I tend to think `const` as immutable vars. isn't that the case?

Reply:
But this is JS so normal rules don't apply 🤣 . In JS a `const` represents the variable assignment, not the contents.

## Fail

https://jsfiddle.net/rcs2zkg9/

```js
const a = "hello"
a = "world"
```

## Pass

https://jsfiddle.net/wgeo6npu/

```js
const a = []
a.push('hello')
a.push('world')
```

This would now fail if I did:

```js
a = []
```

But not if I cleared the original array

```js
a.splice(0, a.length)
```